packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLinks.tsx

```diff
@@ -52,16 +59,16 @@ export const DocumentLinks = ({ appBridge, selectedUrl, onSelectUrl }: DocumentL
             return itemsToExpand;
         }
         for (const document of documentArray) {
-            const pages = await appBridge.getDocumentPagesByDocumentId(document.id);
-            appBridge.getAllDocuments();
+            const pages = await getDocumentPagesByDocumentId(document.id);
+            getAllDocuments();
```

Comment:
I think this call was here by mistake. We can remove it

Reply:
Seems like it, and stupid me didn't saw it, thanks